pipelines/example/measurement_tract_table_selectors_example.yaml

```diff
@@ -0,0 +1,31 @@
+description: Compute metrics from tract catalogs
+tasks:
+  nsrcMeasTract:
+    class: lsst.faro.measurement.TractTableMeasurementTask
+    config:
+      connections.package: info
+      connections.metric: nsrcMeasTract
+      python: |
+        from lsst.faro.base import NumSourcesTask
+        import lsst.faro.utils.selectors as selectors
+        config.measure.retarget(NumSourcesTask)
+  TE3:
```

Comment:
TE3 is for single-visit ellipticity correlations. If this is intended to be a tract level (using object table), this would be TE1.